api/controllers/v1/dochub.ts

```diff
@@ -0,0 +1,10 @@
+import { FastlyConnector } from "../../../src/services/cdn";
+import c from "config";
+import { ConsoleLogger } from "../../../src/services/logger";
+export const UpsertEdgeDictionaryItem = async (event: any = {}): Promise<any> => {
+    const pair = {
+        key: event.detail.fullDocument.name,
+        value: event.detail.fullDocument.url
+    }
+    await new FastlyConnector(new ConsoleLogger).upsertEdgeDictionaryItem(pair, c.get('fastlyDochubMap'), c.get('cdn_creds')['dochub']);
```

Comment:
do we need to `try` & `catch` here, or will any exceptions automagically bubble up into CloudWatch? 

I'm not 100% on our configuration or how much data would be in a failure of this lambda, if we need to debug

Reply:
exceptions will be logged in the cloudwatch. i can add try catcgh